Makefile

```diff
@@ -1,15 +1,14 @@
 .PHONY: web firmware
+USERID ?= \\\#1000
 
 all: 	web firmware
 
 firmware:
-	docker pull golang:1.12-alpine
-	docker run --rm -v $(PWD):/tmp/dev -w /tmp/dev golang:1.12-alpine /bin/sh -c 'apk add make git upx && make -C firmware clean all'
+	docker run --rm -e HOME=/tmp/dev -v $(PWD):/tmp/dev -w /tmp/dev jdevelop/passkeeperbuild:1.12-alpine /bin/sh -c 'make -C firmware clean all'
```

Comment:
jdevelop hardcoded. NG

Reply:
that is the name of docker repo on dockerhub.